docs/tutorials/microsoft365/getting-started-m365.md

```diff
@@ -0,0 +1,205 @@
+
+Set up your M365 account to enable security scanning using Prowler Cloud.
+
+## Requirements
+
+To configure your M365 account, you’ll need:
+
+1. Obtain your `Default Domain` from the Entra ID portal.
+
+2. Access Prowler Cloud and add a new cloud provider `Microsoft 365`.
+
+3. Configure your M365 account:
+
+    3.1 Create the Service Principal app.
+
+    3.2 Grant the required API permissions.
+
+    3.3 Assign the required roles to your user.
+
+    3.4 Retrieve your encrypted password.
+
+4. Add the credentials to Prowler Cloud.
+
+## Step 1: Obtain your Domain
+
+Go to the Entra ID portal, then you can search for `Domain` or go to Identity > Settings > Domain Names.
+
+![Search Domain Names](./img/search-domain-names.png)
+
+Once you are there just look for the `Default Domain` this should be something similar to `YourCompany.onmicrosoft.com`. To ensure that you are picking the correct domain just click on it and verify that the type is `Initial` and you can't delete it.
+
+![Search Default Domain](./img/search-default-domain.png)
+
+---
+
+## Step 2: Access Prowler Cloud
+
+1. Go to [Prowler Cloud](https://cloud.prowler.com/)
+2. Navigate to `Configuration` > `Cloud Providers`
+
+    ![Cloud Providers Page](../img/cloud-providers-page.png)
+
+3. Click on `Add Cloud Provider`
+
+    ![Add a Cloud Provider](../img/add-cloud-provider.png)
+
+4. Select `Microsoft 365`
+
+    ![Select Microsoft 365](./img/select-m365-prowler-cloud.png)
+
+5. Add the Domain ID and an optional alias, then click `Next`
+
+    ![Add Domain ID](./img/add-domain-id.png)
+
+---
+
+## Step 3: Configure your M365 account
+
+
+### Create the Service Principal app
+
+A Service Principal is required to grant Prowler the necessary privileges.
+
+1. Access **Microsoft Entra ID**
+
+    ![Overview of Microsoft Entra ID](./img/microsoft-entra-id.png)
+
+2. Navigate to `Manage` > `App registrations`
+
+    ![App Registration nav](./img/app-registration-menu.png)
+
+3. Click `+ New registration`, complete the form, and click `Register`
+
+    ![New Registration](./img/new-registration.png)
+
+4. Go to `Certificates & secrets` > `+ New client secret`
+
+    ![Certificate & Secrets nav](./img/certificates-and-secrets.png)
+    ![New Client Secret](./img/new-client-secret.png)
+
+5. Fill in the required fields and click `Add`, then copy the generated value
+
+| Value | Description |
+|-------|-------------|
+| Client ID | Application ID |
+| Client Secret | AZURE_CLIENT_SECRET |
+| Tenant ID | Azure Tenant ID |
+
+---
+
+### Grant required API permissions
+
+Assign the following Microsoft Graph permissions:
+
+    - `Directory.Read.All`: Required for all services.
+    - `Policy.Read.All`: Required for all services.
+    - `User.Read` (IMPORTANT: this is set as **delegated**): Required for the sign-in.
+    - `Sites.Read.All`: Required for SharePoint service.
+    - `SharePointTenantSettings.Read.All`: Required for SharePoint service.
```

Comment:
```suggestion
- `Directory.Read.All`: Required for all services.
- `Policy.Read.All`: Required for all services.
- `User.Read` (IMPORTANT: this is set as **delegated**): Required for the sign-in.
- `Sites.Read.All`: Required for SharePoint service.
- `SharePointTenantSettings.Read.All`: Required for SharePoint service.
```

Removing indent will fix the bullet points rendering. Now it render all the block as a big code block.